### PR TITLE
feat(game-level): add bird and bird physics

### LINKED ISSUE #29: Add Physics to Bird Game Page
**Is your feature request related to a problem? Please describe.**
Bird does not have any physics

**Describe the solution you'd like**
For the bird to be able to be thrown around

### LINKED ISSUE #30: Create Ground
**Is your feature request related to a problem? Please describe.**
Create the ground

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Added physics to the bird object
2. Added a ground and physics to the ground object

## Purpose
To have physics in our game

Closes #29
Closes #30

### CODE REVIEW

File: components/Bird.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
 import React from "react";
 import { StyleSheet, View } from "react-native";
 
-const Bird = () => {
+const Bird = ({position, diameter}) => {
 
+  
     return(
-        <View style={styles.bird}/>
+        <View style={[styles.bird, {
```
Comment (dannysanchez559) [inline components/Bird.js:8]: interesting

**Review by connielion — APPROVED** [review]
Looks good so far!

**Review by dannysanchez559 — APPROVED** [review]
very cool stuff

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,7 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+.idea
```
Comment (BlackCubes) [inline .gitignore:11]: What is that 👀

**Pull request merged**